### ISSUE TITLE
Export error with special characters

### ISSUE THREAD

**Issue by Philippe-M** [ClickAndMortar/AdvancedCsvConnectorBundle#36]
Hello,

When a field contains the following character the export fails:
- ® -> registered trademark symbol
- à

By continuing to search I think I have found the cause. If the field is of text type, the presence of a special character prevents the export, but if it is of type text zone then there is no problem for exporting with special characters.

Akeneo 3.2.11 CE / ACCB 1.6.10

**Comment by Philippe-M** [comment]
It's ok with the last version 1.8.5.

Thank

**Comment by Philippe-M** [comment]
Hello,

after updating my version of akeneo 4.0.61 with version 1.8.5 of the plugin I again have the error when I export a text type field containing a special character, an accent for example.

![image](https://user-images.githubusercontent.com/4558101/95089473-79495c00-0724-11eb-977a-9bf95242eb44.png)

what I cannot explain is that on my test server which is in the same version of debian, of php, of mysql I do not have this problem

**---- Solution ----**
I found the reason for the bug.

In my repository I limited this field to 30 characters. In the product sheet of a test product, I filled the field with 30 characters, the last of which being an "é".

If I remove the 30 character limit in the repository then the export goes smoothly.
